mysql-test/include/gap_lock_error_update.inc

```diff
@@ -71,7 +71,6 @@ insert into u1 values (1,2,1,1) on duplicate key update c4=10;
 commit;
 begin;
 select * from u1 where c3=1 and c1 = 1 for update;
---error ER_UNKNOWN_ERROR
 select * from u1 where c3=1 for update;
```

Comment:
But this is not a point select, is it? c3 is only an initial part of an unique column, and as such, this query might return a range of rows